packages/documentation/src/stories/misc/versions.components.tsx

```diff
@@ -0,0 +1,71 @@
+import stylesVersions from '@swisspost/design-system-styles/package.json';
+import componentsVersions from '@swisspost/design-system-components/package.json';
+import internetHeaderVersions from '@swisspost/internet-header/package.json';
+import componentsReactVersions from '@swisspost/design-system-components-react/package.json';
+import documentationVersions from '../../../package.json';
+
+const dependencies = [
+  stylesVersions,
+  componentsVersions,
+  internetHeaderVersions,
+  componentsReactVersions,
+];
```

Comment:
Where is the components-angular (intranet-header) project?

Reply:
It's not a dependency of the documentation as it's not possible to document Angular components (yet).